src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml

```diff
@@ -9,7 +9,7 @@
 	<![CDATA[
 	Add .htaccess scanner (Issue 3972).<br>
 	Modified trace.axd scanner to leverage new AbstractAppFilePlugin component.<br>
-	]]>
+	]]><![CDATA[Add NoSQL scan rule(s) (Issue 3480).]]>

```

Comment:
This change should be added as a line in the original CDATA block (instead of adding a second one).